experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -56,6 +58,43 @@ export const fuzzReducer = withNumberOfEditsCounted(
 			applyTransactionEdit(tree, contents);
 			return state;
 		},
+		reconnect: async (state, operation) => {
+			const { index } = operation;
+			assert(state.containersInfo !== undefined);
+			state.testTreeProvider?.opProcessingController.resumeProcessing();
+			assert(state.testTreeProvider !== undefined);
+			const configProvider = (
+				settings: Record<string, ConfigTypes>,
+			): IConfigProviderBase => ({
+				getRawConfig: (name: string): ConfigTypes => settings[name],
+			});
+			const testContainerConfig: ITestContainerConfig = {
+				loaderProps: {
+					configProvider: configProvider({
+						"Fluid.Container.enableOfflineLoad": true,
+					}),
+				},
+			};
+			const loader = state.testTreeProvider.makeTestLoader(testContainerConfig);
+			const url = state.containersInfo[index].url;
+			const pendingOps = state.containersInfo[index].pendingOps;
+			const loadedContainer = await loader.resolve({ url }, pendingOps);
+			await waitForContainerConnection(loadedContainer, true);
+			state.containersInfo[index].pendingOps = undefined;
+			state.containersInfo[index].url = "";
+		},
+		disconnect: async (state, operation) => {
+			const { index } = operation;
+			assert(state.containersInfo !== undefined);
+			const pauseContainer = state.testTreeProvider?.containers[index];
+			assert(pauseContainer !== undefined);
+			const url = (await pauseContainer.getAbsoluteUrl("")) ?? fail("didn't get url");
+			await state.testTreeProvider?.opProcessingController.pauseProcessing(pauseContainer);
+			const pendingOps = pauseContainer.closeAndGetPendingLocalState();
```

Comment:
this isn't actually the reconnect flow--it looks like it's triggering offline load instead. I'm not sure we have an easy way to incur the reconnect/resubmit flow when using the e2e infra like we've done here. You might be able to do it by forcing connection state change (e.g. using `forceReadOnly`) combined with pausing/resuming the delta connection as you've done, but I'm not sure it's the best approach.

Might be easiest to chat about alternatives in-person... I have a few ideas but depends somewhat on the goals.

Reply:
I had the same thought about this not really being the reconnect flow, but figured that the goal was to test stashed ops/offline load. Is that the intent or should we be doing that separately and adding offline load separately?

Reply:
ideally we test both. it's somewhat awkward that our e2e test infra doesn't allow easy testing of the reconnect+resubmit flow, whereas our mocks don't allow easy testing of the offline load flow. but i'm not sure it's super important that we test them together as part of the same fuzz test.

I'm scheduled for a brown-bag (which keeps getting pushed back due to other meetings taking my timeslot :P) to talk about a generic DDS fuzzing harness I wrote (see work item [here](https://dev.azure.com/fluidframework/internal/_workitems/edit/2280)) that's now ready to use (and is being used in 3-4 other DDSes) and handles reconnect out-of-the-box. It'd be great if SharedTree's fuzz testing was set up to plug nicely into that--it doesn't have to be *all* of the fuzz tests that you write, but it would be one way to work around this issue. I have a lot of opinions in this area though, so as I said might be easier to just chat about it in person.